content/docs/reference-react.md

```diff
@@ -370,9 +370,17 @@ It is documented in our [code splitting guide](/docs/code-splitting.html#reactla
 
 While this is not supported today, in the future we plan to let `Suspense` handle more scenarios such as data fetching. You can read about this in [our roadmap](/blog/2018/11/27/react-16-roadmap.html).
 
->Note:
->
->`React.lazy()` and `<React.Suspense>` are not yet supported by `ReactDOMServer`. This is a known limitation that will be resolved in the future.
+#### `React.Suspense` in SSR
+During SSR trees wrapped in Suspense are rendered synchronously as deeply as possible. If the boundary never suspends 
+then the tree renders as though the boundary were never there. If a component does suspend then we schedule a low 
+priority task to render the fallback instead. If the component unsuspends before we flush the fallback then we send 
+down the actual content and throw away the fallback.
+
+#### `React.Suspense` during hydration
+Suspense boundaries depend on their parent boundaries being hydrated before they can hydrate, but they can hydrate independently from sibling boundaries. Events on a boundary before its hydrated will cause the boundary to hydrate at 
```

Comment:
I was struggling with the "they" references. Does it sound clearer to split this sentence up into two? 
```suggestion
Suspense boundaries depend on their parent boundaries being hydrated before they can hydrate. Suspense boundaries can hydrate independently from sibling boundaries. Events on a boundary before it's hydrated will cause the boundary to hydrate at 
```